docs/06-concepts/11-authentication/05-custom-overrides.md

```diff
@@ -42,6 +46,36 @@ List<String> scopes = extractScopes(token);
 Set<Scope> userScopes = scopes.map((scope) => Scope(scope)).toSet();
 ```
 
+### Handling revoked authentication 
+
+When a user's authentication is revoked, the server must be promptly notified to maintain secure access control. The `authenticationRevoked` event plays a crucial role in closing method streams that the user is no longer authorized to access. To trigger this event, invoke the `session.messages.authenticationRevoked` method.
```

Comment:
Suggestion: Remove adverbs/adjectives and "leaking implementation" details

Two concrete examples just to explain what I'm getting at:
- "promptly" is superfluous info, the user either must or mustn't do it
- Wether the event "plays a crucial role" or not is beside the point (implementation detail), the user just needs to know what event needs to be raised
- "maintain secure access control" is vague, while something explicit like "the server will respect the new changes" is unambiguous 

The following text contains the same info with these things removed:

> When a user's authentication is revoked, the server must be notified to respect the changes (e.g. to close method streams). Invoke the `session.messages.authenticationRevoked` method and raise the appropriate event to notify the server.

Let me know what you think, as it's also a matter of writing style. Personally I prefer to go through technical documentation with as crisp explanations as possible because that makes it easier to find what I'm looking for when searching/skimming.

Reply:
Great suggestion! Simplification is everything. Will use this